src/clj_meowallet/http.clj

```diff
@@ -30,37 +32,68 @@
     (json/generate-string body)
     body))
 
+(defn get-status-from-error
+  [response]
+  (-> (.getMessage response)
+      (str/split #" ")
+      second))
+
 (defn- prepare-error
   "Handles post-response errors"
   [data response]
   (try
-    (-> response
-        (assoc :success false)
-        (assoc :error (str "Error getting " (:url data)))
-        (assoc :requests (inc (:requests data)))
-        (assoc :status (-> response :data :cause))
-        (assoc :body-data (slurp (-> response :data :body))))
+    (cond
+      (instance? java.util.concurrent.TimeoutException response)
```

Comment:
Maybe we should build an aleph wraper to async channels.. Will need this on the main system, here, clanhr...